src/include/homestore/superblk_handler.hpp

```diff
@@ -32,6 +32,30 @@ class superblk {
 
     superblk(const std::string& meta_name = "") { set_name(meta_name); }
 
+    superblk(const superblk&) = delete;
+    superblk& operator=(const superblk&) = delete;
+
+    superblk(superblk&& rhs) noexcept
+         : m_meta_mgr_cookie(rhs.m_meta_mgr_cookie)
+           , m_raw_buf(std::move(rhs.m_raw_buf))
+           , m_sb(rhs.m_sb)
+           , m_metablk_name(std::move(rhs.m_metablk_name)) {
+	rhs.m_meta_mgr_cookie = nullptr;
+	rhs.m_sb = nullptr;
```

Comment:
why this two members is initialized in initialization list and set to nullptr here? 
they are already initialized with nullptr

Reply:
this is move construction